OctoKit/PullRequest.swift

```diff
@@ -323,6 +366,37 @@ enum PullRequestRouter: JSONPostRouter {
                 parameters["maintainer_can_modify"] = (mantainerCanModify ? "true" : "false")
             }
             return parameters
+
+//            func pullRequest(_ session: RequestKitURLSession = URLSession.shared,
+//                             owner: String,
+//                             repository: String,
+//                             title: String,
+//                             head: String,
+//                             head_repo: String?,
+//                             base: String,
+//                             body: String?,
+//                             maintainerCanModify: Bool?,
+//                             draft: Bool?,
+
+        case let .createPullRequest(_, _, _, title, head, headRepo, base, body, mantainerCanModify, draft):
+            var parameters = [
+                "title": title,
+                "head": head,
+                "base": base
+            ]
+            if let headRepo {
+                parameters["head_repo"] = headRepo
+            }
+            if let body {
+                parameters["body"] = body
+            }
+            if let mantainerCanModify {
+                parameters["maintainer_can_modify"] = (mantainerCanModify ? "true" : "false")
+            }
+            if let draft {
+                parameters["draft"] = (draft ? "true" : "false")
```

Comment:
Same here. Just writing it like this should be sufficient.
```suggestion
                parameters["draft"] = draft
```